tests/test_view.py

```diff
@@ -59,4 +60,156 @@ def test_pycon(self):
         self.assertFalse(view.mgr.pycon.python_redirect)
 
 
+def angle_axis(angle_deg, axis):
+    a = axis
+    angle = np.radians(angle_deg)
+
+    cos_angle = np.cos(angle)
+    sin_angle = np.sin(angle)
+
+    rotation = np.identity(3) * cos_angle
+
+    m = np.outer(a, a) * (1 - cos_angle)
+    n = np.array([
+        [0, -a[2], a[1]],
+        [a[2], 0, -a[0]],
+        [-a[1], a[0], 0]
+    ]) * sin_angle
+
+    rotation += m + n
+
+    return rotation
+
+
+@unittest.skip("GUI is not yet available for testing")
+class ViewFPSCameraTC(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
To confirm: do you need corresponding teardown?

Reply:
Thank you for pointing to this. I'm gonna add it